### ISSUE TITLE
IPv6 doesn't work

### ISSUE THREAD

**Issue by lel-amri** [lel-amri/wsgi-lithium#1]
When doing `wsgi-lithium -b :::8000 wsgiref.simple_server:demo_app`, I get :

```
Traceback (most recent call last):
  File "/home/leo/Documents/working/wsgi-lithium/env/bin/wsgi-lithium", line 8, in <module>
    sys.exit(main())
  File "/home/leo/Documents/working/wsgi-lithium/wsgi_lithium/__init__.py", line 60, in main
    with make_server(host, port, app) as httpd:
  File "/home/leo/.pyenv/versions/3.7.16/lib/python3.7/wsgiref/simple_server.py", line 153, in make_server
    server = server_class((host, port), handler_class)
  File "/home/leo/.pyenv/versions/3.7.16/lib/python3.7/socketserver.py", line 452, in __init__
    self.server_bind()
  File "/home/leo/.pyenv/versions/3.7.16/lib/python3.7/wsgiref/simple_server.py", line 50, in server_bind
    HTTPServer.server_bind(self)
  File "/home/leo/.pyenv/versions/3.7.16/lib/python3.7/http/server.py", line 138, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/home/leo/.pyenv/versions/3.7.16/lib/python3.7/socketserver.py", line 466, in server_bind
    self.socket.bind(self.server_address)
socket.gaierror: [Errno -9] Address family for hostname not supported
```

**Comment by lel-amri** [comment]
See https://github.com/python/cpython/issues/64414 and https://github.com/python/cpython/issues/82079.

**Comment by lel-amri** [comment]
Fixed in e1ced06385418c8a62a58fe9495b0c4603d99036, released in version 0.2.0.
